pkg/users/delete-account-dialog.js

```diff
@@ -31,7 +31,7 @@ function DeleteAccountDialogBody({ state, change }) {
     return (
         <Checkbox id="account-confirm-delete-files"
                   label={_("Delete files")}
-                  isChecked={delete_files} onChange={checked => change("delete_files", checked)} />
+                  isChecked={delete_files} onChange={(_event, checked) => change("delete_files", checked)} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230425-151143-725ccf60-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)